dashboard/app/models/user.rb

```diff
@@ -2165,6 +2165,21 @@ def record_soft_delete
       update(state: 'deleted', updated_at: Time.now)
   end
 
+  # Restores all of this user's projects that were soft-deleted after the given time
+  # Called after undestroy
+  private def restore_channels_deleted_after(deleted_at)
+    return unless user_storage_id
+
+    channel_ids = PEGASUS_DB[:storage_apps].
+      where(storage_id: user_storage_id).
+      map(:id)
+
+    PEGASUS_DB[:storage_apps].
+      where(id: channel_ids, state: 'deleted').
+      where(Sequel.lit('updated_at >= ?', deleted_at.localtime)).
+      update(state: 'active', updated_at: Time.now)
```

Comment:
The most confusing thing here: We're getting a UTC Time when we grab the User's `deleted_at` (from dashboard_db), but the `updated_at` storage_apps column contains a local (server?) time.  Both times clearly reported information about their timezone offsets, but I guess comparing them in a Sequel literal isn't clever enough to take the timezone into account.  Using `.localtime` to convert the User's delete time to server time before generating the Sequel literal seems to work.